diff --git a/dist/scripts/lib/build-util.js b/dist/scripts/lib/build-util.js
index 8ad215e82f3f1d72e7393ebf21f987cf8764b3b1..538eb34f38757faaba3bc5f66e91576c0a50f45d 100644
--- a/dist/scripts/lib/build-util.js
+++ b/dist/scripts/lib/build-util.js
@@ -205,6 +205,13 @@ export async function buildPackage(frameworkPackageName, validateManifest, modul
             }
         }
     }
+
+	// Some hacks that need formalizing..
+	const freetypeVendorDir = path.join(packageBaseDir, 'node_modules', '@julusian', 'freetype2', 'vendor')
+	if (fs.existsSync(freetypeVendorDir)) {
+		await fs.rm(freetypeVendorDir, { recursive: true, force: true })
+	}
+
     // Create tgz of the build
     let tgzFile = toSanitizedDirname(`${manifestJson.id}-${manifestJson.version}`);
     if (typeof argv['output'] === 'string') {
