diff --git a/dist/scripts/lib/build-util.js b/dist/scripts/lib/build-util.js
index 3aac91fc3343368da00dd3766741585b1c66ace6..9f7ea508f94c0d76082e397684e6fcc1d2e0e8e4 100644
--- a/dist/scripts/lib/build-util.js
+++ b/dist/scripts/lib/build-util.js
@@ -186,6 +186,14 @@ export async function buildPackage(frameworkPackageName, validateManifest, modul
     }
     await writeLegalArtifacts(packageBaseDir, legalInventory);
     enforceLegalInventory(legalInventory, options);
+
+
+	// Some hacks that need formalizing..
+	const freetypeVendorDir = path.join(packageBaseDir, 'node_modules', '@julusian', 'freetype2', 'vendor')
+	if (fs.existsSync(freetypeVendorDir)) {
+		await fs.rm(freetypeVendorDir, { recursive: true, force: true })
+	}
+
     // Create tgz of the build
     let tgzFile = toSanitizedDirname(`${manifestJson.id}-${manifestJson.version}`);
     if (typeof argv['output'] === 'string') {
